calendar-bundle/src/Resources/contao/models/CalendarEventsModel.php

```diff
@@ -232,7 +232,7 @@ public static function findPublishedByParentAndIdOrAlias($varId, $arrPids, array
 		if (!static::isPreviewMode($arrOptions))
 		{
 			$time = Date::floorToMinute();
-			$arrColumns[] = "($t.start='' OR $t.start<='$time') AND ($t.stop='' OR $t.stop>'" . ($time + 60) . "') AND $t.published='1'";
+			$arrColumns[] = "$t.published='1' AND ($t.start='' OR $t.start<='$time') AND ($t.stop='' OR $t.stop>'" . ($time + 60) . "')";
```

Comment:
The order in the query is not relevant AFAIK.

Reply:
I realized this as well – only after I had changed all the queries. 🙄 I like it more though, because now it is more clear that we are checking the publication status here.